lib/active_merchant/billing/gateways/nmi.rb

```diff
@@ -127,6 +130,7 @@ def scrub(transcript)
         transcript.
           gsub(%r((password=)[^&\n]*), '\1[FILTERED]').
           gsub(%r((ccnumber=)\d+), '\1[FILTERED]').
+          gsub(%r((security_key=)[^&\n]*), '\1[FILTERED]').
```

Comment:
```suggestion
          gsub(%r((password=)[^&\n]*), '\1[FILTERED]').
          gsub(%r((security_key=)[^&\n]*), '\1[FILTERED]').
          gsub(%r((ccnumber=)\d+), '\1[FILTERED]').
```
Not a big deal, but would be nice to keep them grouped conceptually.